File: CuponSmart/src/cuponsmart/modelo/pojo/respuesta/RespuestaCategoria.java
package cuponsmart.modelo.pojo.respuesta;

import cuponsmart.modelo.pojo.entidad.Categoria;
import java.util.List;

public class RespuestaCategoria extends Mensaje{
    private List<Categoria> contenido;
    
    public List<Categoria> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Categoria> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaCategoria.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Categoria

class RespuestaCategoria(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Categoria>? = null
): Mensaje(error, mensaje)